webapp/src/main/webapp/news-details/components/ExoNewsPin.vue

```diff
@@ -19,7 +19,10 @@
       rel="tooltip"
       data-placement="bottom"
       @click="confirmAction">
-      <i :class="[newsPinned ? '' : 'unpinned']" class="uiIconPin"> </i>
+      <v-icon
+        dense
+        :style="broadcastIconStyle"
```

Comment:
we should use a class not a style

Reply:
I think that we should use a style instead of  adding a class that contains only color attribut.